CMakeLists.txt

```diff
@@ -111,7 +111,21 @@ option(BUILD_DOCS "Build documentation (requires doxygen)" OFF)
 option(DEBUG_ERROR_ON_WARNING "For a debug build, error on a compiler# warning" OFF)
 option(BARE_METAL "To configure for bare metal build" OFF)
 option(CUSTOM_MALLOC "Use custom malloc in bare metal builds" OFF)
-
+option(INCLUDE_WHAT_YOU_USE "Run include-what-you-use" ON)
```

Comment:
This should be off by default. Also the variable name is a bit misleading. Should be `RUN_INCLUDE_WHAT_YOU_USE` surely?

Reply:
Ok, turned it off by default and renamed